src/main/index.js

```diff
@@ -14,8 +15,16 @@ class WelcomePage extends React.Component {
     render() {
         return (
             <MuiThemeProvider>
-                <div className="main_header">
-                    <NavBar />
+                <div className="bp--nav-container">
```

Comment:
There's a way that we can reduce the nested-ness of our app toolbar using flex instead of grid. We should log an issue to look that up and refactor in the future.

Reply:
Sounds good!